load/UpdateDBPTextFilesets.py

```diff
@@ -84,14 +84,24 @@ def invokeBiblePublisher(self, inp, fullFilesetPath):
 
 		return None
 
```

Comment:
delete methods invokeBiblePublisher, getBiblePublisherVerses, removeSpecialChar, verse2StringInt,
this will require a change to updateFilesetTextPlain to also remove the call to getBiblePublisherVerses. 
Re-testing will be required if the BiblePublisher verses were used to insert into the DB

Reply:
> delete methods invokeBiblePublisher, getBiblePublisherVerses, removeSpecialChar, verse2StringInt, this will require a change to updateFilesetTextPlain to also remove the call to getBiblePublisherVerses. Re-testing will be required if the BiblePublisher verses were used to insert into the DB

@bradflood
- I have removed the method: `invokeBiblePublisher`
- I think that for now, we can't remove the method: `verseString2Int` because it is validating that the verseStart and verseEnd values are integers. When we handle the case where either the verseStart or verseEnd can be strings, we could remove this method. You can see the comment about that [here](https://github.com/faithcomesbyhearing/dbp-etl/pull/91/files#diff-38516f84a6c11ce60e5c2399f71fb8e10c8fffbc654c8e51082685269a35aa11R175)
- I have removed `removeSpecialChar` because I have implemented this method into sofria-client.
- I have renamed the method: `getBiblePublisherVerses` to `getBibleVersesFromSqlite` and it can not be removed because this method is used from load/UpdateDBPTextFilesets.py line [114](https://github.com/faithcomesbyhearing/dbp-etl/pull/91/files#diff-38516f84a6c11ce60e5c2399f71fb8e10c8fffbc654c8e51082685269a35aa11R114) to update the `bible_verses` DBP entity.